modules/client/test/mocks/aragon-sdk-ipfs.ts

```diff
@@ -0,0 +1,19 @@
+export const mockedIPFSClient = {
+  nodeInfo: jest.fn().mockResolvedValue(true),
+  add: jest.fn().mockResolvedValue({
+    hash: "QmXhJawTJ3PkoKMyF3a4D89zybAHjpcGivkb7F1NkHAjpo",
+  }),
+  cat: jest.fn().mockImplementation(async () => Buffer.from("{}")),
+  __proto__: {},
```

Comment:
We seem to be mocking `add` only once, it feels like there should be many more uses in the testing?  Should `pin` be mocked as well?

Reply:
what do you mean with once? this mock is the general mock. the other instances is overriding the default mock. Especially needed for `cat`
Yep `pin` would make sense, once we call it somewhere

Reply:
Oh, so that single mocking of `cat` affects everywhere? I thought the mock somehow was file-scoped :thinking: 

That may bring unexpected behavior in unrelated files where this line of code is not visible. Do we have a way to "enable" and "disable" these mocks, independently on each file?

I would expect having that at the `before()` and `after()` blocks of each test suite... otherwise, the tests where the `cat` mocking is done are polluting the rest.

Same if this applies to `cat`